examples/with-stitches/stitches.config.js

```diff
@@ -1,6 +1,6 @@
-import { createCss } from '@stitches/react'
+import { createStitches } from '@stitches/react'
 
-export const { css, styled, global, getCssString } = createCss({
+export const { css, styled, globalCss, getCssText } = createStitches({
```

Comment:
Can we also export `theme`, `createTheme`, and `config`? Thanks!

Reply:
@peduarte It's done 👍 

Thanks for all the work you and your team have put in on Stitches and Radux UI. They make a great combo 😃